GameBoard/Startup.cs

```diff
@@ -58,7 +58,7 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            if (env.IsDevelopment())
+            if (env.IsDevelopment() || env.IsStaging())
```

Comment:
Staging should resemble production in all ways, so it shouldn't have a development exception page.

Reply:
Fixed